cs/ccxt/base/Exchange.Time.cs

```diff
@@ -183,4 +183,87 @@ public object ymd(object ts, object infix = null)
         }
         return timestamp;
     }
+
+    public object convertExpireDate(object date)
+    {
+        // parse YYMMDD to timestamp
+        object year = slice(date, 0, 2);
+        object month = slice(date, 2, 4);
+        object day = slice(date, 4, 6);
+        object reconstructedDate = add(add(add(add(add(add("20", year), "-"), month), "-"), day), "T00:00:00Z");
+        return reconstructedDate;
+    }
+
+    public virtual object convertExpireDateToMarketIdDate(object date)
```

Comment:
this shouldn't be here, because this has "ccxt-related" logic

Reply:
so should it be on the base exchange class?